migrations/1609107624484_create-node-memberships.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable camelcase */
+
+exports.shorthands = undefined;
+
+exports.up = pgm => {
+  pgm.createTable("memberships", {
+    id: "id",
+    node_id: {type: "integer", references: "nodes(id)", notNull: true},
+    member_id: {type: "integer", references: "members(id)", notNull: true},
+  });
```

Comment:
Thinking about this some more -- we'll probably want to extend this at some point to store apartment, notes, etc. Maybe even device id and port number? It would be good to have a status too, so when someone moves out we can set it to inactive rather than deleting the whole record.

Reply:
That sounds like a reasonable future direction to me.